docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+### Options
+
+Miscellaneous options for building an Android app.
+
+- **Android package format**
+
+  Either `apk` or `bundle`, which packages the Android application as an APK file or Android App Bundle, respectively. This can be set for individually for both Debug and Release modes.
+
+  App Bundles are the latest format for Android release builds that are intended for submission on Google Play.
+
+  The default value is `apk`.
+
+  When `bundle` is selected, other MSBuild properties are set:
+
+  - `AndroidUseAapt2` is set to `True`.
+  - `AndroidUseApkSigner` is set to `False`.
+  - `AndroidCreatePackagePerAbi` is set to `False`.
+
+- **Fast deployment (debug mode only)**
+
+  When enabled, deploys the app faster than normal to the target device. This process speeds up the build/deploy/debug cycle because the package isn't reinstalled when only assemblies are changed. Only the updated assemblies are resynchronized to the target device.
+
+  This is enabled by default.
+
+- **Generate per ABI**
+
+  When enabled, generates one Android package (apk) per selected Application Binary Interface (ABI). This is disabled by default.
+
+- **Use incremental packaging**
+
+  When enabled, uses the incremental Android packaging system (aapt2). This is enabled by default.
+
+- **Multi-dex**
+
+  When enabled, allows the Android build system to use multidex. The default is disabled.
+
+- **Code shrinker**
+
+  Selects the code shrinker to use.
+
+  - `ProGuard` (default) is considered the legacy code shrinker.
+  - `r8` is the next-generation tool which converts Java byte code to optimized dex code.
+
+- **Uncompressed resources**
+
+  Leaves the specified resource extensions uncompressed. Separate extensions with a semicolon `;`. For example: `.mp3;.dll;.png`.
+
+- **Developer instrumentation**
+
+  When enabled, developer instrumentation is provided for debugging and profiling. This can be set for individually for both Debug and Release modes.
+
+  The default is enabled for Debug builds.
+
+- **Debugger**
+
+  Selects which debugger to use. The default is `.NET (Xamarin)`, which is used for managed code. The C++ debugger can be selected to debug native libraries used by the app.
+
+- **AOT**
+
+  Enables Ahead-of-Time (AOT) compilation. This can be set for individually for both Debug and Release modes.
+
+  The default is enabled for Release builds.
+
+- **LLVM**
+
+  Enables the LLVM optimizing compiler. The default is disabled.
+
+- **Startup Tracing**
+
+  Enables startup tracing. This can be set for individually for both Debug and Release modes.
+
+  The default is enabled for Release builds.
+
+- **Garbage Collection**
+
+  When enabled, uses the concurrent garbage collector. Defaults to enabled.
+
+- **Enable trimming**
+
+  When enabled, trims the application during publishing. This can be set for individually for both Debug and Release modes.
```

Comment:
```suggestion
  When enabled, trims the application during publishing. This can be set for individually for both Debug and Release modes. For more information, see [Trim self-contained deployments and executables](/dotnet/core/deploying/trimming/trim-self-contained).
```